### DISCUSSION TITLE
Error when install with scoop on w11

### DISCUSSION THREAD

**Discussion by pwsandoval** [sharkdp/bat#2476] [Q&A] [answered]
Hello!

I have used scoop before to install starship and winfetch with no problems (I use windows terminal, windows 11 and powershell 7.3.2).

![scoop list](https://user-images.githubusercontent.com/15174704/218570385-670f3441-80e4-4a96-8107-07a8fd394766.png)

however today when installing bat I got the following error:

![scoop bat](https://user-images.githubusercontent.com/15174704/218570424-90484391-41a3-4168-bf72-e87efad5bc39.png)

As it looks, the installation is finished, and when testing I can use it:

![bat ok](https://user-images.githubusercontent.com/15174704/218570466-a247a1b2-ad10-4ba5-b914-bcb1aef313f1.png)

So I have the question of whether the error will affect any option or feature, and if there is any way to correct it?

Thanks for your time.

**Comment by sharkdp** [comment, marked as answer]
Please report this on the scoop issue tracker. We are not maintaining the scoop installation scripts.
